data/brands/shop/convenience.json

```diff
@@ -5798,6 +5798,22 @@
         "name:zh": "超市发",
         "shop": "convenience"
       }
+    },
+    {
+      "displayName": "十足",
+      "id": "shizu-zhejiang-cn",
+      "locationSet": {"include": ["cn"]},
+      "tags": {
+        "brand": "十足",
+        "brand:en": "Shizu",
+        "brand:zh": "十足",
+        "brand:wikidata": "Q116763971",
+        "name": "十足",
+        "name:en": "Shizu",
+        "name:zh": "十足",
+        "website": "https://www.cnshizu.com",
```

Comment:
> Note for brands, that common tags with additional details such as "website" or "facebook" is not accepted directly. Instead a tag referencing the brand's [wikidata](https://www.wikidata.org/) entry ("[brand:wikidata](https://wiki.openstreetmap.org/wiki/Key:brand:wikidata)") containing those details can be included. If no such exists, consider [adding it](https://www.wikidata.org/wiki/Special:NewItem).

This was mentioned in https://github.com/osmlab/name-suggestion-index/wiki/Contributing#tags-required, I haven't found this before, sorry, and I think you can add it in wikidata as P856 (I've done)